gravitee-node-container/src/test/java/io/gravitee/node/container/spring/env/WatchablePropertyResolverSubscriberTest.java

```diff
@@ -0,0 +1,25 @@
+package io.gravitee.node.container.spring.env;
+
+import io.reactivex.rxjava3.core.Flowable;
+import io.reactivex.rxjava3.functions.Supplier;
+import io.reactivex.rxjava3.subscribers.TestSubscriber;
+import java.util.concurrent.TimeUnit;
+import org.junit.Test;
+import org.junit.jupiter.api.Assertions;
+
+public class WatchablePropertyResolverSubscriberTest {
+
+    @Test
+    public void shouldRepeatFlowableWhenItsCompleted() throws Exception {
+        int secondsDelay = 1;
+        Supplier<Flowable<String>> flowableSupplier = () -> Flowable.just("1");
+        TestSubscriber<String> test = new AbstractGraviteePropertySource.FlowableRepeater<>(flowableSupplier, secondsDelay)
+            .asFlowable()
+            .test();
+
+        test.await(3, TimeUnit.SECONDS);
```

Comment:
The test looks good, however, using TestScheduler would be better than using await.

Reply:
sure, I wasn't aware of such tool. I fixed.